Visualize logical lanes

myemural opened the pull request:
This PR is for visualizing logical lanes.
Only lanes rendered:
![Screenshot from 2025-06-30 17-15-44](https://github.com/user-attachments/assets/d45fbdfc-b909-4ac1-9484-0310f5e038af)

Only logical lanes rendered:
![Screenshot from 2025-06-30 17-16-07](https://github.com/user-attachments/assets/3f2038f6-b694-4832-ac46-4b88e76f8e24)

Both of them rendered:
![Screenshot from 2025-06-30 17-16-25](https://github.com/user-attachments/assets/d191b753-98c4-45c6-b8c8-3d0dae7514ac)

jdsika commented:
The switch will disable the lanes and then only show "logical lanes"?

myemural commented:
> The switch will disable the lanes and then only show "logical lanes"?

Yes, but the problem is this change doesn't happen immediately. As I understood from the Lichtblick settings logic, these settings are not triggering re-render mechanism. It can be also a bug or a feature.

jdsika commented:
It is already notes here: https://github.com/lichtblick-suite/lichtblick/issues/503

myemural commented:
> It is already notes here: [lichtblick-suite/lichtblick#503](https://github.com/lichtblick-suite/lichtblick/issues/503)

OK then. I fixed the conflicts and rebased with main branch. You can check again.

myemural commented:
@jdsika This is not intended behavior but it can be related to this issue also: https://github.com/lichtblick-suite/lichtblick/issues/503

Or it can be a different Lichtblick issue. Because when I leaped to previous or next frames from time bar, changes are applying correctly. But when I used "Play/Pause", "Seek Backward" and "Seek Forward" buttons, it is not updating correctly. I checked also the deletion mechanism on our extension code, but I couldn't see any wrong thing.

jdsika commented:
Maybe we can quickly discuss it today. I think you are using a bool to capture the state but maybe you need a Tri-state (enum) to make sure that you newly deactivate the lanes in order to add the last lane IDs to delete the scene entities?
